README.md

```diff
@@ -1,16 +1,49 @@
-# Title
+# JavaScript Fundamentals: Scope Chain Lab
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+1.  Identify the difference between const and let
+2.  Identify how functions determine scope in Javascript
 
 ## Introduction
 
-## SWBAT 1
+Scuber's drivers are beginning to take some notes down about their customers.
+Use your knowledge of scope to access and change some of these variables.
 
-## SWBAT 2
+## Instructions
 
-## Conclusion
+A word of warning: In this lab, you will practice doing some bad things. For
+example, we'll ask you to declare variables in global scope, and write functions
+that break down. We do this because, by testing limits and exposing the inner
+workings of a language, you can better understand it, and better know what to
+avoid in the future.
 
-## Resources
+Ok, these are your tasks:
+
+- customerName: Declare a variable in global scope called customerName using the
+  `var` keyword.
+
+- upperCaseCustomerName: Write a function that accesses that global customerName
+  variable, and uppercases it.
+
+- setBestCustomer: Write a function that when called, declares a variable called
+  `bestCustomer` in global scope and assigns it to be `'not bob'`. (Poor bob.)
+  Also, poor us, declaring a global variable from inside a function is one of
+  those things we would never want to do normally, but is good for us to explore
+  right now.
+
+- overWriteBestCustomer: See the consequences of declaring a variable in global
```

Comment:
Update this overWriteBestCustomer variable based on what we call the other one. Also make sure to tick.